Source/WebCore/inspector/agents/InspectorDOMAgent.cpp

```diff
@@ -648,6 +649,32 @@ Inspector::Protocol::ErrorStringOr<void> InspectorDOMAgent::requestChildNodes(In
     return { };
 }
 
+Inspector::Protocol::ErrorStringOr<Ref<JSON::ArrayOf<Inspector::Protocol::DOM::NodeId>>> InspectorDOMAgent::requestAssignedNodes(Inspector::Protocol::DOM::NodeId nodeId)
```

Comment:
You can use the newer `Inspector::CommandResult<>` in place of `ErrorStringOr<>`